math/src/msm.rs

```diff
@@ -1,157 +0,0 @@
-use crate::cyclic_group::IsCyclicBilinearGroup;
```

Comment:
Removed this file since we organize differently now in a separate `mod msm` for both this implementation, and the new one.